dist/testbed-support/makefile.iotlab.single.inc.mk

```diff
@@ -48,6 +48,11 @@ ifeq (,$(IOTLAB_NODE))
   $(error)
 endif
 
+ifeq (auto-ssh,$(IOTLAB_NODE))
+  $(warning IOTLAB_NODE=auto-ssh is deprecated, use IOTLAB_NODE=auto instead)
+  override IOTLAB_NODE := auto
```

Comment:
Is override needed? I would be needed only if is set as a MAKE argument.

Reply:
Yes, IOTLAB_NODE is already overridden in master. Otherwise it won't work with a command like `make IOTLAB_NODE=auto`